src/Iterator/HydratingArrayIterator.php

```diff
@@ -25,8 +25,6 @@ class HydratingArrayIterator extends HydratingIteratorIterator
     protected $prototype;
 
     /**
-     * @param HydratorInterface $hydrator
-     * @param array $data
```

Comment:
All over the patch, iterator, array, iterable, generator types MUST be documented, even if just with `mixed[]`